EventListener/SerializerListener.php

```diff
@@ -5,18 +5,18 @@
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\Serializer\Exception\ExceptionInterface as BaseSerializerException;
-use Symfony\Component\Serializer\SerializerInterface;
+use Xabbuh\XApi\Serializer\SerializerRegistryInterface;
 
 /**
  * @author Christian Flothmann <christian.flothmann@xabbuh.de>
  */
 class SerializerListener
 {
-    private $serializer;
+    private $serializerRegistry;
 
-    public function __construct(SerializerInterface $serializer)
+    public function __construct(SerializerRegistryInterface $serializerRegistry)
```

Comment:
I suggest to inject `StatementSerializerInterface` instances instead. IMO there is not much use for the registry in the bundle's code.